lib/native/auth.ts

```diff
@@ -128,11 +129,13 @@ export type AwsSigningConfig = crt_native.AwsSigningConfig;
 export async function aws_sign_request(request: HttpRequest, config: AwsSigningConfig): Promise<HttpRequest> {
     return new Promise((resolve, reject) => {
         try {
+            /* TODO: if the body of request has not fully loaded, it will lead to a endless loop. 
+             * User should set the config to prevent this endless loop for now */
```

Comment:
trivial: delete? It's a "TODO", but I'm not sure what this is saying we should do?

Reply:
Yeah... It's a little bit weird... The idea is to make some notes for future reference to avoid the endless loop... May make it as a Note or something?